Archipelago.HollowKnight/Archipelago.cs

```diff
@@ -285,8 +285,26 @@ public ConnectionDetails OnSaveGlobal()
             {
                 ServerUrl = MenuSettings.ServerUrl,
                 ServerPort = MenuSettings.ServerPort,
-                SlotName = MenuSettings.SlotName
+                SlotName = MenuSettings.SlotName,
+                AlwaysShowItems = MenuSettings.AlwaysShowItems,
             };
         }
+
+        //#region IMenuMod
```

Comment:
I'm not a believer in `#region` especially for an implementation this small.

Reply:
Fair enough! It has been removed.